extension/source/QuillPage/components/SetNicknamePanel.tsx

```diff
@@ -7,30 +7,33 @@ const PasswordCreationPanel: React.FunctionComponent<{
   onComplete: () => void;
 }> = ({ onComplete }) => (
   <>
-    <div className="instructions-text">
-      <h3>It&apos;s time to create your first wallet!</h3>
-      <p>
+    <div className="mb-10">
+      <div className="font-bold">
+        It&apos;s time to create your first wallet!
+      </div>
+      <span>
         Let&apos;s give it a nickname so that you can easily identify it when
         you have more wallets.
-      </p>
-    </div>
-    <div>
-      <input type="text" placeholder="Nickname" />
+      </span>
     </div>
-    <div className="explainer-box">
-      Setting a nickname is optional but recommended.
-    </div>
-    <div>
-      <div style={{ display: 'inline-block' }}>
-        <Button
-          onPress={onComplete}
-          className="btn-primary"
-          icon={<ArrowRight className="icon-md" />}
-        >
-          Create wallet
-        </Button>
+    <div className="h-40">
+      <input type="text" placeholder="Nickname" className="input" />
+      <div className="bg-grey-200 p-4 mt-4 text-[10pt] rounded-md flex gap-4">
+        <Info className="icon-md text-blue-500 mt-1" />
+        <div className="align-text-top">
+          Setting a nickname is optional but recommended.
+        </div>
       </div>
     </div>
+    <div className="py-24 float-right">
+      <Button
+        onPress={onComplete}
+        className="btn-primary w-32"
+        icon={<ArrowRight className="icon-md" />} // TODO: Where is svg?
```

Comment:
Where is the SVG :smiley: I assume we are using font icons now and would prefer to use SVGs?

Reply:
```suggestion
        icon={<ArrowRight className="icon-md" />}
```

Reply:
I added this and removed it in other PRs but I think weird merge things happened.